doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-rest-api-case-admin-ref.adoc

```diff
@@ -0,0 +1,237 @@
+      "case-definition-id": "itorders.orderhardware",
+      "container-id": "itorders_1.0.0-SNAPSHOT",
+      "case-started-at": 1540474204152,
+      "case-completed-at": null,
+      "case-completion-msg": "",
+      "case-sla-compliance": 0,
+      "case-sla-due-date": null,
+      "case-file": null,
+      "case-milestones": null,
+      "case-stages": null,
+      "case-roles": null
+    },
+    {
+      "case-id": "IT-0000000007",
+      "case-description": "Order for IT hardware",
+      "case-owner": "baAdmin",
+      "case-status": 1,
+      "case-definition-id": "itorders.orderhardware",
+      "container-id": "itorders_1.0.0-SNAPSHOT",
+      "case-started-at": 1540486490427,
+      "case-completed-at": null,
+      "case-completion-msg": "",
+      "case-sla-compliance": 0,
+      "case-sla-due-date": null,
+      "case-file": null,
+      "case-milestones": null,
+      "case-stages": null,
+      "case-roles": null
+    },
+    {
+      "case-id": "IT-0000000008",
+      "case-description": "Order for IT hardware",
+      "case-owner": "baAdmin",
+      "case-status": 1,
+      "case-definition-id": "itorders.orderhardware",
+      "container-id": "itorders_1.0.0-SNAPSHOT3",
+      "case-started-at": 1540491516562,
+      "case-completed-at": null,
+      "case-completion-msg": "",
+      "case-sla-compliance": 0,
+      "case-sla-due-date": null,
+      "case-file": null,
+      "case-milestones": null,
+      "case-stages": null,
+      "case-roles": null
+    },
+    {
+      "case-id": "IT-0000000009",
+      "case-description": "Order for IT hardware",
+      "case-owner": "baAdmin",
+      "case-status": 1,
+      "case-definition-id": "itorders.orderhardware",
+      "container-id": "itorders_1.0.0-SNAPSHOT4",
+      "case-started-at": 1540492038543,
+      "case-completed-at": null,
+      "case-completion-msg": "",
+      "case-sla-compliance": 0,
+      "case-sla-due-date": null,
+      "case-file": null,
+      "case-milestones": null,
+      "case-stages": null,
+      "case-roles": null
+    },
+    {
+      "case-id": "IT-0000000011",
+      "case-description": "Order for IT hardware",
+      "case-owner": "baAdmin",
+      "case-status": 1,
+      "case-definition-id": "itorders.orderhardware",
+      "container-id": "itorders_1.0.0-SNAPSHOT6",
+      "case-started-at": 1540496734746,
+      "case-completed-at": null,
+      "case-completion-msg": "",
+      "case-sla-compliance": 0,
+      "case-sla-due-date": null,
+      "case-file": null,
+      "case-milestones": null,
+      "case-stages": null,
+      "case-roles": null
+    }
+  ]
+}
+----
+
+== [PUT] admin/containers/{containerId}/cases/instances/{caseId}
+
+Migrates a specified case instance to another KIE container and case definition.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`containerId`
+|ID of the KIE container where the case instance resides
```

Comment:
IBM SG: reside v
Use “reside” to refer to someone’s place of residence, not as a location for inanimate objects.

Consider...where the case instance is located?

Reply:
Haha! I was hoping no one would notice :) I looked at IBM SG (our own SG now I guess :) ) when I before I started using it and saw the same, and even switched to "is located", but found that is technically should be "in which X is locate" instead of "where X located". But now that I think of it, same with "resides", ha! So I'll change to "is located" throughout. Thanks Emily.